rulebook.tex

```diff
@@ -380,193 +435,196 @@ \subsection{Executive Committee 2015--2019}
-\textit{robot insertion area}, and \textit{Base, Storage and Delivery
-  Stations}, are located. We will refer to the side with positive
-coordinates on the x-axis as the (primary) half of team cyan, and the
-side with negative coordinates on the x-axis as the primary half of
-team magenta.
-
-The robot insertion areas (gray area on \reffig{fig:competition-area})
-are outside of the main game area. All 14 machines, 7 per team, are
-placed within the factory area as stated in
-\reffig{fig:competition-area}. Machines have two kind of sides: active
-(wide, where the robot interacts) and ignored (narrow) edges.  The
-constraints for positioning of the MPS are given in
-\refsec{sec:machine-swapping}.
-
-The distribution and alignment of all machines is axially symmetrical
-to the y-axis. Thus, each team has similar conditions on both halves of
-the competition field. Both teams have an exclusive set of 7 out of
-the 14 machines each. Machines from one team can also be located on the
-primary side of the opposing team. We will later discuss the concept
-of the production machine distribution for each team in detail.
-
-The center coordinates and the size of the zones in which the machines
-are placed can be seen in \reffig{fig:competition-area}. The zone
-names are structured as a team prefix (``C-'' for cyan, ``M-'' for
-magenta), Z to distinguish zone names, and a grid coordinate. For
-example, The zone C-Z23 is the zone on the cyan primary half, which is
-the second along the X-axis and the third along the Y-axis. Zones are
-only a means to position machines. They will not be physically
-represented or visible on the field. They cannot be used for any other
-purpose.
-
-Note that this specific field layout is an example only, accounting
-for symmetry and avoiding clustering of machine access nodes and
-unapproachable machines. However, the actual distribution and
-alignment of machines on the competition area will change before the
-actual game starts and will be different for each game. Thus, teams
+The competition area of the \ac{RCLL} has a rectangular shape. The field is
+divided in square zones of \SI{1 x 1}{\metre}. %chktex 29
+The field is partially surrounded (at least 50\% but not more than 70\%)
+by wall elements of at least \SI{0.5}{\metre} height.
+Different machines are placed within the area, the constraints for
+positioning them are given in \refsec{sec:machine-swapping}.
+The \ac{RCLL} supports two kind of field layouts:
+Firstly, a symmetrical one
+with aspect ratio 1:2 intended for two teams to compete simultaneously
+on the same area.
+Secondly, a field with a squared layout that is intended for tasks,
+where only one team is on the field at a time.
+Fields also contain designated robot insertion areas, that are used as a
+starting position for the robots when a game starts or a robot is inserted
+into a running game. The usage of these areas is restricted to these purposes,
+unless stated otherwise.
```

Comment:
I am not aware of a situation where it is allowed to use an insertion zone, are there any? If not, I would shorten the sentences here to just: a robot is only allowed to enter the playing field via the insertion zone. 
Specially in this years competitions we where driving there, since we did not had the wall in front of it, But I saw that you updated the challanges layout - was this intended to have that change?

Reply:
Without DS we deliver workpieces to the insertion zone in the challenge track.

Regarding the rules regarding walls: I noticed that this is not properly described in the rulebook, i will add a clearification